doc/memos/rdm0.md

```diff
@@ -0,0 +1,126 @@
+RIOT Maintainers Community (RIOT-MC)  
+Developer Memo: 0  
+Author(s): Emmanuel Baccelli  
+Date: December 2016
```

Comment:
See https://github.com/RIOT-OS/RIOT/pull/6285#discussion_r94622819. This does not parse always parse as in your file. To be on the save side I would suggest to make this a list.